src/main/java/org/sunbird/portal/department/model/Context.java

```diff
@@ -0,0 +1,99 @@
+package org.sunbird.portal.department.model;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "channel", "pdata", "env", "cdata", "rollup" })
+
+public class Context {
```

Comment:
Need to move this to another new package which should be separate for telemetry related classes.